### PR TITLE
PROD321-5985 - Homepage Promos

### PR DESCRIPTION
**Pull request by frankynztein**
## Overview

Updated copy ans styles for Homepage promos. I asked for the icon in Promo 1. This will be updated as soon as a I get that file.

Connects [PROD321-5985](https://321ignitionproj.atlassian.net/browse/PROD321-5985)

 ### Demo

image

### CODE REVIEW

File: _site/scss/home.scss [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
-  color: $white;
-  letter-spacing: 0.25px;
-}
-
-.icon-glow {
-  color: $bright;
-  text-shadow: 0 0 3px $bright;
+.ad {
+  .banner-first, .banner-second {
+    background-color: $secondary;
```
Comment (GabrielaDiazB) [inline _site/scss/home.scss:4]: We have a variable for the first banner `$first-ad-banner-background`

File: _site/scss/home.scss [hunk 2]
```diff
@@ -1,24 +1,37 @@
...
+  }
+  
+  .banner-third {
+    .fi-star {
+      font-size: 28px;
+    }
```
Comment (GabrielaDiazB) [inline _site/scss/home.scss:10]: Please use helpers before overwriting the styles. The ads are freestyle you can use the `font-size-14` for this.

File: _site/scss/home.scss [hunk 3]
```diff
@@ -1,24 +1,37 @@
...
+  
+    .text-one {
+      font-size: 16px;
+    }
```
Comment (GabrielaDiazB) [inline _site/scss/home.scss:14]: Same here, use helpers before overwriting

File: _site/templates/banners/ads.md [hunk 4]
```diff
@@ -3,48 +3,47 @@ ads: [
     html:
       '<div class="banner-first h-100 d-flex flex-column justify-content-between align-items-center py-4 px-3 position-relative">
         <div class="w-100">
-          <p class="text-one h3-reversed mb-0">What car can your FICO<sup>®</sup> score get you?</p>
-          <img class="w-50 pl-3 mb-3" src="https://img.321ignition.io/dealer-images/p7TEdZilBfy3V3afNZJo/home/credit-graph.svg" alt="white sedan thumbnail">
+          <p class="text-one h3-reversed mb-0 danger-underline underline-center">Sell us your car for top dollar</p>
+          <img class="w-50 pl-3 mb-3" src="" alt="">
```
Comment (GabrielaDiazB) [inline _site/templates/banners/ads.md:7]: If you are not going to use an image here, then please remove the tag

**Review by GabrielaDiazB — CHANGES_REQUESTED** [review]

File: _site/scss/home.scss (hunk 1 above)
Comment (frankynztein) [reply _site/scss/home.scss:4]: Done.

File: _site/scss/home.scss (hunk 3 above)
Comment (frankynztein) [reply _site/scss/home.scss]: Done.

File: _site/scss/home.scss (hunk 2 above)
Comment (frankynztein) [reply _site/scss/home.scss]: Done.

File: _site/templates/banners/ads.md (hunk 4 above)
Comment (frankynztein) [reply _site/templates/banners/ads.md:7]: I'm actually waiting for the image/icon (not sure) that goes there.

**Review by GabrielaDiazB — APPROVED** [review]

**Pull request merged**